Automattic/wp-calypso
# Sidebar: rename "Plan" section to "Upgrades"

Now that site-level purchase management has been added and a "Plan" section created, we have the redundant `Plan > Plans > My Plan` navigation. To house all purchase-related navigation, this PR does the following:

- renames the "Plan" section to "Upgrades"
- moves the plan indicator to the "Plans" navigation item
- adds a (duplicate for now) link to the domain management screen
- renames the "Billing" item to "Purchases"
- within the "Purchases" site-level and account-level screens:
  - updates the formatted header to "Purchases"
  - renames the "Purchases" section nav item to "Active Upgrades"
- Updates the fallback for the unified domains project (see: https://github.com/Automattic/jetpack/pull/18093)

See: pbOQVh-KN-p2

Before | After
------------ | -------------
![before](https://user-images.githubusercontent.com/942359/102799169-23c63800-4380-11eb-8234-ba3909154bbf.png) | ![after](https://user-images.githubusercontent.com/942359/102799182-288aec00-4380-11eb-835b-8239df2522ec.png)


**To test:**
- visit Calypso
- verify that the "Upgrades" section is shown in the sidebar and matches the screenshot above
- verify that the links inside the section correctly navigate to '/plans', '/purchases', and '/domains'
- verify that the sectionNav, formattedHeader, and various pageTitles inside the Purchases screen are accurate
- make sure that the tests and e2es pass
